docs/full-node/run-a-full-terra-node/updates-and-additional.mdx

```diff
@@ -20,7 +20,7 @@ rm ~/.terra/config/addrbook.json
 terrad unsafe-reset-all
 ```
 
-Your node is now in a pristine state, keeping the original `priv_validator.json` and `config.toml`. If you had any sentry nodes or full nodes set up before, your node will still try to connect to them but may fail if they haven't also been upgraded.
+Your node is now in a pristine state, keeping the original `priv_validator.json` and `config.toml`. If you had any sentry nodes or full nodes set up before, your node will still try to connect to them, but may fail if they haven't also been upgraded.
```

Comment:
This sounds a little misleading.
Terrad tendermint unsafe reset all will delete the private key, and this will lead to a tombstoned validator.